Test fails with
> Error in codegen for ComputedBuffer(name='buf2', ...

Skip it as the fix has been reverted, see https://github.com/pytorch/pytorch/pull/183843

Author: Alexander Grund (TU Dresden)
diff --git a/test/inductor/test_select_algorithm.py b/test/inductor/test_select_algorithm.py
--- a/test/inductor/test_select_algorithm.py
+++ b/test/inductor/test_select_algorithm.py
@@ -1246,9 +1246,7 @@ class TestTemplateRender(TestCase):
                 kernels[0]
             )
 
-    @unittest.skipIf(
-        TEST_WITH_ROCM or TEST_XPU, "https://github.com/pytorch/pytorch/issues/179959"
-    )
+    @unittest.skip("https://github.com/pytorch/pytorch/issues/179959")
     @requires_gpu()
     @requires_triton()
     @config.patch(cuda_backend="triton")
